### ISSUE TITLE
ARM64 Support

### ISSUE THREAD

**Issue by NikoGrano** [sentry-kubernetes/charts#960]
Chart currently has no support for servers running ARM64.

Currently failing images:
- `yandex/clickhouse-server`
- `getsentry/snuba:23.7.2` (Please see [this](https://github.com/getsentry/self-hosted/issues/1585#issuecomment-1678303029))
- `bitnami/zookeeper:3.8.0`

## Clickhouse
Yandex has been real bad giving support for multiarch images lately... It could be replaced with an image from `bitnami/clickhouse`, however first support for the arm was introduced in version [22.8.13-debian-11-r9](https://hub.docker.com/layers/bitnami/clickhouse/22.8.13-debian-11-r9/images/sha256-ff4b66b60c50b59e4b260ea96eeaea6ea2b9772763f2337f4e8cf5b9b63726a8?context=explore).

## Snuba
Needs official support from sentry. Please see getsentry/self-hosted/issues/1585

## Zookeeper
Bitnami introduced support for arm on release [3.8.1-debian-11-r9](https://hub.docker.com/layers/bitnami/zookeeper/3.8.1-debian-11-r9/images/sha256-ded358311789a1506cb3e90ad6313a98b07676e7f42f744cd1e9f69aceacab4b?context=explore). Should upgrade to this patch version.

## TL;DR

I won't work on this, but I am just reporting the findings. Anybody feel free to pick this up or close this issue if nothing happens. Also solving this issue without bad workarounds would need an official sentry team to give arm support for snuba.

**Comment by alekitto** [comment]
Hi, I'm currently running sentry with (a sligthly modified) version of this chart on a kube cluster hosted on Oracle Cloud with Ampere nodes (ARM). Currently I've replaced the failing images this way:

>     * `yandex/clickhouse-server`

`knatnetwork/clickhouse-server:21.8.3.44-lts` => is the nearest clickhouse image supporting ARM without a full recompilation. 

>     * `getsentry/snuba:23.7.2`

It's been six months already that I recompile the sentry images (sentry, snuba, relay and now vroom) for multi arch (amd64, arm64). Every time a new release is published I recompile these images with almost no modifications (relay and sentry need some edits on the Dockerfile, while snuba and vroom do not). If someone want to try them, I write them down here:

- `public.ecr.aws/alekitto/sentry:23.7.2`
- `public.ecr.aws/alekitto/snuba:23.7.2`
- `public.ecr.aws/alekitto/sentry-relay:23.7.2`
- `public.ecr.aws/alekitto/vroom:23.7.2`

Obviously you can change the tag to match your version (all tags from `23.2.0` should be available, if something is missing and you need that specific version, just let me know).

>     * `bitnami/zookeeper:3.8.0`

Upgrading to `3.8.1` should be safe, anyway I personally use an external redpanda deployment which in my case is faster, less memory hungry and does not need zookeeper

**Comment by Mokto** [comment]
This issue is stale because it has been open for 30 days with no activity.

**Comment by Mokto** [comment]
This issue was closed because it has been inactive for 14 days since being marked as stale.
